backend/src/main/resources/db-migrations/common/v0-[common]-schema-init.sql

```diff
@@ -1,33 +1,83 @@
-CREATE TABLE alert_type (
-	id VARCHAR(64) NOT NULL,
+CREATE TABLE `user` (
+	`id` VARCHAR(64) NOT NULL,
 
-	code VARCHAR(64) NOT NULL,
-	description VARCHAR(256),
+	`email` VARCHAR(256),
+	`email_verified` BOOLEAN,
 
 	-- audit fields
-	created_by VARCHAR(64) NOT NULL,
-	created_date TIMESTAMP NOT NULL,
-	last_modified_by VARCHAR(64),
-	last_modified_date TIMESTAMP,
+	`created_by` VARCHAR(64) NOT NULL,
+	`created_date` TIMESTAMP WITH TIME ZONE NOT NULL,
+	`last_modified_by` VARCHAR(64),
+	`last_modified_date` TIMESTAMP WITH TIME ZONE,
 
-	CONSTRAINT pk_status_code PRIMARY KEY (id)
+	CONSTRAINT `pk_user` PRIMARY KEY (`id`)
 );
 
-CREATE TABLE subscription (
-	id VARCHAR(64) NOT NULL,
-	user_id VARCHAR(9) NOT NULL,
-	email VARCHAR(50) NOT NULL,
-	registered BOOLEAN,
-	subscribed BOOLEAN,
-	preferred_language BIGINT,
-	alert_type_id VARCHAR(64) NOT NULL,
+CREATE INDEX `ix_user_email` ON `user` (`email`);
+
+
+create TABLE `user_attribute` (
+	`id` VARCHAR(64) NOT NULL,
+
+	`name` VARCHAR(256) NOT NULL,
+	`value` VARCHAR(2048),
+	`user_id` VARCHAR(64) NOT NULL,
+
+	-- audit fields
+	`created_by` VARCHAR(64) NOT NULL,
+	`created_date` TIMESTAMP WITH TIME ZONE NOT NULL,
+	`last_modified_by` VARCHAR(64),
+	`last_modified_date` TIMESTAMP WITH TIME ZONE,
+
+	CONSTRAINT `pk_user_attribute` PRIMARY KEY (`id`),
+	CONSTRAINT `fk_user` FOREIGN KEY (`user_id`) REFERENCES `user` (`id`),
+	CONSTRAINT `uq_user_attribute` UNIQUE (`name`, `user_id`)
+);
+
+CREATE INDEX `ix_user_attribute_user_id` on `user_attribute` (`user_id`);
+CREATE INDEX `ix_user_attribute_name_value` ON `user_attribute` (`name`, `value`);
+
+
+
+CREATE TABLE `alert_type` (
+	`id` VARCHAR(64) NOT NULL,
+
+	`code` VARCHAR(64) NOT NULL,
+	`description` VARCHAR(256),
 
 	-- audit fields
-	created_by VARCHAR(64) NOT NULL,
-	created_date TIMESTAMP WITH TIME ZONE NOT NULL,
-	last_modified_by VARCHAR(64),
-	last_modified_date TIMESTAMP WITH TIME ZONE,
-
-	CONSTRAINT pk_subscription PRIMARY KEY (id),
-	CONSTRAINT fk_subscription_alert_type FOREIGN KEY (alert_type_id) REFERENCES alert_type(id)
-);
\ No newline at end of file
+	`created_by` VARCHAR(64) NOT NULL,
+	`created_date` TIMESTAMP WITH TIME ZONE NOT NULL,
+	`last_modified_by` VARCHAR(64),
+	`last_modified_date` TIMESTAMP WITH TIME ZONE,
+
+	CONSTRAINT `pk_status_code` PRIMARY KEY (`id`)
```

Comment:
it is my mistake. please update this to pk_alert_type_id for me

Reply:
Updated [here](https://github.com/DTS-STN/canadian-dental-care-plan/pull/1389/files#diff-500b9845965d76e18de5b8c1564c83d71728d542d830e552ea026a804db26175R54)